Sources/Basics/HTTPClient.swift

```diff
@@ -34,9 +35,10 @@ public struct HTTPClient {
     private static var hostsErrorsLock = Lock()
     private static var hostsErrors = [String: [Date]]()
 
-    public init(configuration: HTTPClientConfiguration = .init(), handler: @escaping Handler) {
+    public init(configuration: HTTPClientConfiguration = .init(), handler: Handler? = nil) {
         self.configuration = configuration
-        self.underlying = handler
+        // FIXME: inject platform specific implementation here
```

Comment:
this will be addressed in later PR by creating a non-Darwin version and conditionalizing the code